docs/howtos/passing-protobuf-data-over-ffi.md

```diff
@@ -196,6 +196,12 @@ follow the examples of the other steps it takes.
         }
     ```
 
+If you make any changes to the `.proto` file and want regenerate your local kotlin files, you can use:
+```
+./gradlew :projectname:generateDebugProto
```

Comment:
```suggestion
./gradlew generateDebugProto
```
will work!

Reply:
I'd be nice if we had a gradle target that found all tasks of `GenerateProtobufTask` type (extract, debug/release builds etc.) in our project and run them. Maybe you can create a follow-up issue?

Reply:
Nice!